src/main/java/io/confluent/idesidecar/restapi/kafkarest/ClusterManagerImpl.java

```diff
@@ -13,7 +13,7 @@
 import static io.confluent.idesidecar.restapi.util.MutinyUtil.uniStage;
 import static io.confluent.idesidecar.restapi.util.RequestHeadersConstants.CONNECTION_ID_HEADER;
 
-import io.confluent.idesidecar.restapi.cache.AdminClients;
+import io.confluent.idesidecar.restapi.clients.AdminClients;
 import io.confluent.idesidecar.restapi.cache.ClusterCache;
```

Comment:
Hmm, something is definitely wrong with our build, because the sorting order (alphabetical, `cache` before `clients`) should have been caught. :-(
Let's fix this now, and deal with the build problem separately.
```suggestion
import io.confluent.idesidecar.restapi.cache.ClusterCache;
import io.confluent.idesidecar.restapi.clients.AdminClients;
```
Same for all the other files where this change is made.

Reply:
Actually, only this change needs to be fixed -- the only other files that import something from `cache` are correct.